content/operator/euskotren/index.de.md

```diff
@@ -0,0 +1,91 @@
+---
+draft: false
+title: 'Euskotren'
+country:
+  - 'spain'
+  - 'france'
+operator: 'euskotren'
+---
+
+{{< wip >}}
+
+Im Norden Spaniens zwischen der französischen Grenze bei Hendaye und der Metropole Bilbao verbindet ein Netz der baskischen Eisenbahn die malerischen Küstenorte und das gebirgige Hinterland. Euskotren ist Eigentum der baskischen Regionalregierung.
+
+## Zusammenfassung
+
+- Vor Ort wird bei Vorlage des FIP Ausweises kostenfrei ein Ticket ausgestellt, das 7 Tage gültig ist und zur Nutzung des gesamten Euskotren Netzes mit Ausnahme von Bussen berechtigt.[^1]
+- Mit dem Ticket können die Züge, die U-Bahnlinie L3 in Bilbao sowie Straßenbahnen und die Standseilbahn "Funicular de Larreineta" genutzt werden.
+- Das Ticket kann nach Ablauf auch mehrfach pro Jahr neu ausgestellt werden. Die sonst übliche Limitierung bei FIP Freifahrtscheinen gilt bei Euskotren nicht.
+
+## Gültigkeit FIP Tickets
+
+FIP Freifahrtsschein: ✅, kann vor Ort ausgestellt werden.
+
+## Zugkategorien und Reservierungen
+
```

Comment:
Hier wäre eine Zusammenfassung zu Reservierungen gut. Z.B bei Euskotren gibt es keine Reservierungen in den Zügen